solvebio/cli/main.py

```diff
@@ -249,6 +249,15 @@ class SolveArgumentParser(argparse.ArgumentParser):
                     "create any folders.",
                     "action": "store_true",
                 },
+                {
+                    "flags": "--archive-folder",
+                    "help": "Path to archive files that already exist. "
```

Comment:
Path to an existing folder? or will the folder be auto-created?

Reply:
Currently, it auto-creates. We could make it fail so that users have to be a bit careful about this and that there aren't multiple archive folders mistakenly generated. What do you think?

Reply:
Probably fine to auto create